### PR TITLE
12.0 mig membership extension

### PR DESCRIPTION
**Pull request by amcor**
cc @guadaltech

### CODE REVIEW

File: membership_extension/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
 # Copyright 2017 David Vidal <david.vidal@tecnativa.com>
 # Copyright 2017 Luis M. Ontalba <luis.martinez@tecnativa.com>
 # Copyright 2017-2018 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2019 Alberto Martín Cortada <alberto.martin@guadaltech.es>
+
```
Comment (chienandalu) [inline membership_extension/__manifest__.py]: Thanks @amcor !

Being picky, you should not put your copyright for this migration (no code or functionality was added) :slightly_smiling_face: 

Also don't add unnecessary blank lines

Comment (amcor) [reply membership_extension/__manifest__.py]: No problem! in the contributors readme it's correct ?

Comment (chienandalu) [reply membership_extension/__manifest__.py]: Sure! :slightly_smiling_face:

**Comment by chienandalu** [comment]
Check runbot warnings as well: https://runbot3.odoo-community.org/runbot/static/build/3363072-64-76bdd6/logs/job_20_test_all.txt

**Comment by chienandalu** [comment]
Tests are failing. I'd say it's just on date formats on comparisons (remember that on v12 date fields return a datetime object), so it should not be hard to fix

**Comment by amcor** [comment]
@chienandalu all green !! Thanks David.

File: membership_extension/data/membership_category_data.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
```
Comment (nikul-serpentcs) [inline membership_extension/data/membership_category_data.xml]: @amcor `<odoo noupdate="1">`

File: membership_extension/demo/membership_category_demo.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+<data>
```
Comment (nikul-serpentcs) [inline membership_extension/demo/membership_category_demo.xml]: @amcor no need to `<data>`

File: membership_extension/demo/product_template_demo.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+<data>
```
Comment (nikul-serpentcs) [inline membership_extension/demo/product_template_demo.xml]: same here

File: membership_extension/views/product_template_view.xml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+<record model="ir.ui.view" id="membership_product_search_form_view">
+    <field name="name">Add membership category field</field>
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="membership.membership_product_search_form_view" />
+    <field name="arch" type="xml">
+        <field name="categ_id" position="after">
+            <field name="membership_category_id" />
+        </field>
+        <xpath expr="//search//group//filter[1]" position="after">
+            <filter name="Membership category" domain="[]"
```
Comment (nikul-serpentcs) [inline membership_extension/views/product_template_view.xml]: @amcor `name="membership_category" string="Membership category"`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
netpicking

**Comment by manuelcalerosolis** [comment]
Is this module ready to work in version 12 of odoo?

**Comment by pedrobaeza** [comment]
@manuelcalerosolis you can do a review of it for speeding up its merge:

https://odoo-community.org/page/review

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Some squash made before merging

**Pull request merged**